practices/code-review.md

```diff
@@ -0,0 +1,34 @@
+# Code Reviews
+
+Code Review, or Peer Code Review, is the act of consciously and systematically checking the code or your colleagues for mistakes and errors. It has been repeatedly shown to accelerate and streamline the process of software development.
+
+We manage code review via **pull requests**, which we communicate in the `#front-end-pr` Slack channel, and manage in GitHub. All discussion happens in GitHub comments. We leave it to the requester to merge their own changes -- we never merge someone else's work!
+
+## Why?
+
+- PR reviews are cross-team
+    - allows greater visibility on other teams work
+    - allows for standardisation of methodology across teams
+    - allows more junior members of staff to learn best practices quicker
+    - allows EVERYONE to learn from their colleagues
+    - facilitates people moving to other teams as they are more familiar with the codebase
+- Ensure that FED's within each team are coding to the same guidelines
```

Comment:
We might want to use "frontend developers" instead of "FEDs" since this is a public repo and "FED" is a bit of an idiomatic term to this org.